src/repositories/FeedFetcherRepository.ts

```diff
@@ -26,11 +23,8 @@ export class FeedFetcherRepository {
       PolygonIOStockPrice,
       PolygonIOCryptoPrice,
       PolygonIOCurrencySnapshotGrams,
-      OnChainData,
       YearnVaultTokenPrice,
-      OptionsPrice,
       UniswapPriceFetcher,
-      RandomNumber,
       TWAPGasPrice: evmTWAPGasPriceFetcher,
```

Comment:
@DZariusz these fetchers are not following the fetcher interface. I just remove it from the repository since are not being used. Would you prefer that I fix them and try to adapt them to follow the interface?

Reply:
ok, but let's not remove the files.

Reply:
they will remain there